src/test/java/web/component/Label.java

```diff
@@ -16,4 +18,9 @@ public Label(By locator) {
     public Label(WebElement element) {
         super(element);
     }
+
+    public String gatherPriceByNumber(int numberOfButton) {
```

Comment:
this method is a custom method that is working not for all the labels, so it should be excluded from the Label class and implemented in the corresponding screen-class